Document where to make changes to the navigation sidebar (#240)
srobo/docs: README.md

[install-ruby]: https://www.ruby-lang.org/en/documentation/installation/
[raise-a-pr]: https://github.com/srobo/docs/pull/new/master

### Navigation Sidebar

The docs navigation sidebar is generated from `_data/sidebar_tree.yaml` as part

> of the build process. This file is manually updated in order that we can

Reviewer: ```suggestion
of the build process. This file is manually updated in order so that we can
```

Author: I think the grammar here is already correct?

Reviewer 2: Honestly the use of "in order" while discussing how the file is ordered is confusing.
```suggestion
of the build process. This file is manually updated so that we can
```
